charts/restate-helm/values.yaml

```diff
@@ -29,6 +29,10 @@ securityContext:
 logging:
   env_filter: info
 
+nodeName: $(POD_NAME)
+# Use the stable network identifier (dns resolvable) provided by the stateful set
+advertisedAddress: http://$(RESTATE_NODE_NAME).$(SERVICE_NAME):5122
```

Comment:
Shouldn't this be `${POD_NAME}.${SERIVCE_NAME}` by default in case that I change the node name? When would I change this value?

Reply:
Good point thanks. You would change this address if you wanted to run restate cross region and it necessitated a different dns name - this one only works inside the cluster.